src/main/java/Eliz/Eliz.java

```diff
@@ -0,0 +1,331 @@
+        File f = new File(filePath); //helps to create a file for the give file path
+        Scanner s = new Scanner(f); //create scanner using file as a source
+        int taskCounter = 0;
+        Task input;
+        while (s.hasNext()) { //while the file is not empty
+            String line = s.nextLine();
+            String[] arrayOfTaskComponents = line.split("|", 3);
+            arrayOfTaskComponents[2] = arrayOfTaskComponents[2].substring(1);
+            if (arrayOfTaskComponents[0].contains("T")) {
+                input = new Todo(arrayOfTaskComponents[2]);
+                if (arrayOfTaskComponents[1].contains("X")) {
+                    input.setAsDone();
+                }
+            } else if (arrayOfTaskComponents[0].contains("D")) {
+                input = new Deadline(arrayOfTaskComponents[2]);
+            } else if (arrayOfTaskComponents[0].contains("E")) {
+                input = new Event(arrayOfTaskComponents[2]);
+            } else {
+                return; //error
+            }
+            tasks.add(input);
+            taskCounter++;
+        }
+//        if (breakTaskNames[0].equalsIgnoreCase("todo") || breakTaskNames[0].equalsIgnoreCase("deadline")
+//        || breakTaskNames[0].equalsIgnoreCase("event")) {
+//                /** add line to todo, deadline, or event by creating the respective object */
+//                Task t = createTask(line);
+//                tasks.add(taskCounter, t);
+//                taskCounter++;
+//                System.out.println("Got it. I've added this task ");
+//                System.out.println(t);
+//                System.out.println("Now you have " + taskCounter + " tasks in the list.");
+//        } else {
+//            if (line.equalsIgnoreCase("list")) { //check if action is t
+//                //echo or print tasks
+//                printTasks(tasks);
+//            } else if (line.contains("mark")) { //to check if todos are marked
+//                Eliz.markOrUnmark(line, tasks);
+//            } else if (line.contains("delete")) {
+//                Eliz.delete(line, tasks);
+//            } else {
+//                System.out.println("OOPS!!! I'm sorry but I do not understand what you mean");
+//                throw new ElizException();
+//            }
+//        }
+//        if (!line.equalsIgnoreCase("list") && !line.contains("mark") && breakTaskNames.length < 2) {
+//            System.out.println("OOPS!!! The description of a " + breakTaskNames[0] + " cannot be empty.");
+//            throw new ElizException();
+//        }
+    }
+
+    public static void main(String[] args) {
+        /** Key Definitions */
+        String line;
+        ArrayList<Task> tasks = new ArrayList<>();
+        int taskCounter = 0;
+        Scanner in = new Scanner(System.in);
+        botIntroduction(); //calls the introduction of the bot
+        line = in.nextLine();
+        String filePath = "./task_list.txt";
```

Comment:
You could define this file path as a [constant](https://se-education.org/guides/conventions/java/basic.html#naming) if it is not being further modified anywhere.